Dockerfile.rhel8

```
@@ -0,0 +1,30 @@
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.13 AS builder
+WORKDIR /go/src/github.com/openshift/builder
+COPY . .
+RUN hack/build.sh
+
+FROM registry.svc.ci.openshift.org/ocp/4.0:base
```

Is this currently a ubi-8 based image, or @yselkowitz are you using a `BuildConfig` or OpenShift CI to substitute this with the UBI8 image?

If we want to run this Dockerfile "as is", perhaps this should be replaced with:
```suggestion
FROM registry.access.redhat.com/ubi8/ubi:latest
```

---

Sorry, my focus has been on CI and ART substitution, where the FROMs are replaced anyway.  iiuc 4.0:base is a placeholder anyway, but it should probably be base-8 instead.  WIll fix accordingly.